dist/COVID-VACCINE-TRIAL-schema.json

```diff
@@ -0,0 +1,330 @@
+      "properties": {
+        "view:exposureRiskText": {
+          "type": "object",
+          "properties": {}
+        }
+      }
+    },
+    "residentsInHome": {
+      "type": "string",
+      "enum": [
+        "ONE_TWO",
+        "THREE_FIVE",
+        "SIX_TEN",
+        "MORE_THAN_TEN"
+      ]
+    },
+    "healthHistory": {
+      "type": "object",
+      "properties": {
+        "ALLERGY_VACCINE": {
+          "type": "boolean"
+        },
+        "AUTOIMMUNE_DISEASE": {
+          "type": "boolean"
+        },
+        "CANCER": {
+          "type": "boolean"
+        },
+        "DIABETES": {
+          "type": "boolean"
+        },
+        "HEART_DISEASE": {
+          "type": "boolean"
+        },
+        "HIGH_BLOOD_PRESSURE": {
+          "type": "boolean"
+        },
+        "IMMUNOCOMPROMISED": {
+          "type": "boolean"
+        },
+        "KIDNEY_LIVER_DISEASE": {
+          "type": "boolean"
+        },
+        "LUNG_DISEASE": {
+          "type": "boolean"
+        },
+        "STROKE": {
+          "type": "boolean"
+        },
+        "ANOTHER_SERIOUS_CHRONIC_ILLNESS": {
+          "type": "boolean"
+        }
+      }
+    },
+    "diagnosed": {
+      "type": "boolean"
+    },
+    "closeContactPositive": {
+      "type": "string",
+      "enum": [
+        "YES",
+        "NO",
+        "UNSURE"
+      ]
+    },
+    "hospitalized": {
+      "type": "boolean"
+    },
+    "smokeOrVape": {
+      "type": "boolean"
+    },
+    "employmentStatus": {
+      "type": "object",
+      "properties": {
+        "EMPLOYED_HOME": {
+          "type": "boolean"
+        },
+        "EMPLOYED_OUTSIDE_OF_HOME": {
+          "type": "boolean"
+        },
+        "FRONTLINE_WORKER": {
+          "type": "boolean"
+        },
+        "FURLOUGHED_UNEMPLOYED": {
+          "type": "boolean"
+        },
+        "RETIRED": {
+          "type": "boolean"
+        },
+        "STUDENT": {
+          "type": "boolean"
+        },
+        "NONE_OF_ABOVE": {
+          "type": "boolean"
+        }
+      }
+    },
+    "transportation": {
+      "type": "object",
+      "properties": {
+        "CAR": {
+          "type": "boolean"
+        },
+        "FREQUENT_AIR_TRAVEL": {
+          "type": "boolean"
+        },
+        "PUBLIC_TRANSPORT": {
+          "type": "boolean"
+        },
+        "WALK_BIKE": {
+          "type": "boolean"
+        },
+        "WORK_FROM_HOME": {
+          "type": "boolean"
+        },
+        "NONE_OF_ABOVE": {
+          "type": "boolean"
+        }
+      }
+    },
+    "closeContact": {
+      "type": "string",
+      "enum": [
+        "ZERO",
+        "ONE_TEN",
+        "ELEVEN_THIRTY",
+        "THIRTYONE_FIFTY",
+        "MORE_THAN_FIFTY"
+      ]
+    },
+    "contactHeaderText": {
+      "type": "object",
+      "properties": {
+        "view:contactText": {
+          "type": "object",
+          "properties": {}
+        }
+      }
+    },
+    "zipCode": {
```

Comment:
https://github.com/department-of-veterans-affairs/vets-json-schema/blob/master/src/common/definitions.js#L54

Reply:
@mchelen-gov I recall this one giving me a difficult time reusing common def for zipcode without address but I will check it out and see if we can use a common def for zipCode

Reply:
@mchelen-gov pushed change to use common def for `usaPostalCode`.  Good catch thanks for the eyes!